code/controllers/subsystem/mobs.dm

```diff
@@ -7,7 +7,7 @@ SUBSYSTEM_DEF(mobs)
 	var/list/currentrun = list()
 	var/static/list/clients_by_zlevel[][]
 	var/static/list/cubemonkeys = list()
-	var/cubemonkeycap = 64
+	var/cubemonkeycap = CONFIG_GET(number/monkeycap)
```

Comment:
why is travis whining about this

Reply:
You need to toss it under initialize rather than the definition of the datum